### DISCUSSION TITLE
how do I redo Prepare System

### DISCUSSION THREAD

**Discussion by tekjock** [paukstelis/octoprint_deploy#54] [General]
I chose the wrong option, and now when I rerun it, I cannot do Prepare System  again. I have deleted the  octoprint_deploy folder, but when I add it again, there is no option to prepare system

**Comment by paukstelis** [comment]
don't delete octoprint_deploy folder for starters!
`git clone https://github.com/paukstelis/octoprint_deploy`
`sudo octoprint_deploy/octoprint_deploy.sh remove`
